src/main/java/seedu/address/MainApp.java

```diff
@@ -108,11 +110,11 @@ protected Config initConfig(Path configFilePath) {
         configFilePathUsed = Config.DEFAULT_CONFIG_FILE;
 
         if (configFilePath != null) {
-            logger.info("Custom Config file specified " + configFilePath);
+            logger.fine("Custom Config file specified " + configFilePath);
             configFilePathUsed = configFilePath;
         }
 
-        logger.info("Using config file : " + configFilePathUsed);
+        logger.config("Using config file : " + configFilePathUsed);
```

Comment:
Let's not use the config level. severe -> warning -> info -> fine -> finer -> finest is much cleaner hierarchy with config jumping in the middle and muddling things. Besides, it seems to be Java specific thing which the other levels has better transferability to other languages.